webapp-django/crashstats/api/tests/test_views.py

```diff
@@ -851,3 +859,101 @@ def test_telemetry_has_crash(self, client):
                 u'elasticsearch_crash': False,
             }
         )
+
+
+class TestMultipleStringField:
+    """Test the MultipleStringField class."""
+
+    def test_empty_list_required(self):
+        """If a field is required, an empty list is a validation error."""
+        field = MultipleStringField()
+        with pytest.raises(ValidationError):
+            field.clean([])
+
+    def test_empty_list_optional(self):
+        """If a field is optional, an empty list is valid."""
+        assert MultipleStringField(required=False).clean([]) == []
+
+    def test_good_argument(self):
+        """A list with one string arguments is valid."""
+        assert MultipleStringField().clean(['one']) == ['one']
+
+    def test_null_arg(self):
+        """A embedded null character is a validation error."""
+        field = MultipleStringField()
+        value = "Embeded_Null_\x00"
+        with pytest.raises(ValidationError):
+            field.clean([value])
+
+
+API_MODEL_NAMES = [
+    "Bugs",
+    "NewSignatures",
+    "NoOp",
+    "ProcessedCrash",
+    "RawCrash",
+    "Reprocessing",
+    "SignatureFirstDate",
+    "SignaturesByBugs",
+    "SuperSearch",
+    "SuperSearchFields",
+    "SuperSearchUnredacted",
+    "UnredactedCrash",
+    "VersionString",
```

Comment:
You're using " vs. ' here. have a bug for running black on Antenna. Maybe we should run black on Socorro, too.